### PR TITLE
445: More Entity Validations

### PR DESCRIPTION
**Pull request by ericsorenson**

### CODE REVIEW

File: shared/src/business/entities/cases/Case.js [hunk 1]
```diff
@@ -339,8 +339,14 @@ function Case(rawCase, { applicationContext }) {
 joiValidationDecorator(
   Case,
   joi.object().keys({
-    associatedJudge: joi.string().required(),
-    blocked: joi.boolean().optional(),
+    associatedJudge: joi
+      .string()
+      .required()
+      .description('Defaults to Chielf Judge.'),
```
Comment (kkoskelin) [inline shared/src/business/entities/cases/Case.js]: Typo, should be `Chief`

File: shared/src/business/entities/cases/Case.js [hunk 2]
```diff
@@ -449,7 +459,7 @@ joiValidationDecorator(
       })
       .optional()
       .description(
-        'If this Case is consolidated, this is the ID of the lead Case.',
+        'If this Case is consolidated, this is the ID of the lead Case. It is the lowest number in the consolidated group.',
```
Comment (kkoskelin) [inline shared/src/business/entities/cases/Case.js]: `It is the ID of the case with the lowest docket number...` ?

**Review by kkoskelin — CHANGES_REQUESTED** [review]
A few things to be updated.

File: shared/src/business/entities/cases/Case.js (hunk 2 above)
Comment (ericsorenson) [reply shared/src/business/entities/cases/Case.js]: Updated!

File: shared/src/business/entities/cases/Case.js (hunk 1 above)
Comment (ericsorenson) [reply shared/src/business/entities/cases/Case.js]: Updated!

**Review by kkoskelin — APPROVED** [review]

**Review by kkoskelin — APPROVED** [review]

**Pull request merged**